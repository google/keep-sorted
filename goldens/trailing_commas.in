Trailing comma except for last
  keep-sorted-test start
  3,
  1,
  2
  keep-sorted-test end


Trailing comma and a comment
TODO: https://github.com/google/keep-sorted/issues/33 - Fix this
  keep-sorted-test start
  3, # three
  1,
  2
  keep-sorted-test end

Trailing comma except for last, last has a comment
TODO: https://github.com/google/keep-sorted/issues/33 - Fix this
  keep-sorted-test start
  3,
  1,
  2  # two
  keep-sorted-test end

Trailing comma and a comment, last has a comment
TODO: https://github.com/google/keep-sorted/issues/33 - Fix this
  keep-sorted-test start
  3, # three
  1,
  2  # two
  keep-sorted-test end
