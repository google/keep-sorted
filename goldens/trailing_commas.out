Trailing comma except for last
  keep-sorted-test start
  1,
  2,
  3
  keep-sorted-test end


Trailing comma and a comment
TODO: https://github.com/google/keep-sorted/issues/33 - Fix this
  keep-sorted-test start
  1,
  2
  3, # three
  keep-sorted-test end

Trailing comma except for last, last has a comment
TODO: https://github.com/google/keep-sorted/issues/33 - Fix this
  keep-sorted-test start
  1,
  2  # two,
  3
  keep-sorted-test end

Trailing comma and a comment, last has a comment
TODO: https://github.com/google/keep-sorted/issues/33 - Fix this
  keep-sorted-test start
  1,
  2  # two
  3, # three
  keep-sorted-test end
